### ISSUE TITLE
Darktable creates linear Rec. 2020 profile with whitepoint xyY in XYZ tag

### ISSUE THREAD

**Issue by sboukortt** [darktable-org/darktable#4343]
**To Reproduce**
1. Export an image with profile “linear Rec2020 RGB”
2. Extract the ICC profile from the exported image (e.g. with `convert image.png profile.icc`)
3. Run `cd-iccdump` from colord or `displaycal-profile-info` from DisplayCAL on the profile

**Expected behavior**
The whitepoint tag looks like it does with the Rec. 2020 profile from ArgyllCMS, i.e.:

~~~
tag 05:
  sig   'wtpt' [0x77747074]
  size  20
  type  'XYZ ' [0x58595a20]
XYZ:
  X:0.950455 Y:1.000000 Z:1.089050
~~~

**Actual behavior**
The whitepoint looks like this:

~~~
tag 02:
  sig   'wtpt' [0x77747074]
  size  20
  type  'XYZ ' [0x58595a20]
XYZ:
  X:0.312698 Y:0.328995 Z:1.000000
~~~

Which is clearly a xyY chromaticity triplet (an unadapted one, at that), not an XYZ one.

**Platform:**
 - Darktable Version: 3.0.0
 - OS: Arch Linux

**Comment by aurelienpierre** [comment]
See https://github.com/darktable-org/darktable/pull/4108

**Comment by sboukortt** [comment]
Oh, great, thank you. Somehow I didn’t see that one.

**Comment by jenshannoschwalm** [comment]
Fixed via #4745
